src/status_im/contexts/wallet/share_address/view.cljs

```diff
@@ -63,7 +63,14 @@
                                                        {:url         qr-url
                                                         :port        (rf/sub [:mediaserver/port])
                                                         :qr-size     qr-size
-                                                        :error-level :highest})]
+                                                        :error-level :highest})
+            ;; status is either :share or :receive. when it's :share, the title will be :t/share-address
+            ;; and when it's :receive, the title will be :t/receive
```

Comment:
I think this comment is covered already by the case statement, so it's sort of unnecessary. At least to me, it's clear what are the possible values for `status` and which translation will be used.

Reply:
Done